cmd/serve/serve.go

```diff
@@ -204,50 +207,50 @@ func NewCommand(
 			//v1beta2
 			// wire together resource handling
 			resource_repo := resourcerepo.New(db)
-			resource_controller := resourcesctl.New(resource_repo, inventoryresources_repo, authorizer, eventingManager, "notifications", log.With(logger, "subsystem", "notificationsintegrations_controller"), storageConfig.Options.DisablePersistence, listenManager)
+			resource_controller := resourcesctl.New(resource_repo, inventoryresources_repo, authorizer, eventingManager, "notifications", log.With(logger, "subsystem", "notificationsintegrations_controller"), storageConfig.Options.DisablePersistence, listenManager, inventoryConsumer)
```

Comment:
Slight issue here. In latest code, the consumer is not fully instantiated until its own go routine is called because in order to enable retries in polling we have to be able to restart the consumer without killing the entire app. See https://github.com/project-kessel/inventory-api/blob/0a299d340e65bb25f124510c536594c001aa8290/cmd/serve/serve.go#L259 

So this call in new code would be passing a nil value around. Instead perhaps we could pass consumerConfig value similar to how storageConfig is done here for persistance

something like
```
consumerConfig.Options.ReadAfterWriteEnabled
consumerConfig.Options.ReadAfterWriteAllowlist
```